src/client/pages/OfferNew/Introduction/Sidebar/CancellationOptions.tsx

```diff
@@ -13,12 +13,12 @@ const HandleSwitchingWrapper = styled.div`
   padding: 0 0.25rem;
 `
 
-const HandleSwitchingLabel = styled.label<{ isClickable: boolean }>`
+const HandleSwitchingLabel = styled.label<{ isEnable: boolean }>`
```

Comment:
Typo 🙊 
```suggestion
const HandleSwitchingLabel = styled.label<{ isEnabled: boolean }>`
```

But also, would it make more sense to switch it and make it `isDisabled` instead of `isEnabled`?  Since it looks like that's more what we want to use it for (like when we pass it as `isLoading` and `disabled`).